drivers/vmwarevsphere/vsphere.go

```diff
@@ -314,22 +314,29 @@ func (d *Driver) Create() error {
 		return err
 	}
 
+	if d.CreationType == "" {
```

Comment:
Can this case be added to the switch statement?

Reply:
its already caught by the default case, this will give us a different error message though if it was accidently passed empty. i would say add what was passed as CreationType in the error in the default cause and this is just as good as adding this additiona check